src/Sulu/Component/Serializer/ArraySerializationVisitor.php

```diff
@@ -11,12 +11,12 @@
 
 namespace Sulu\Component\Serializer;
 
-use JMS\Serializer\GenericSerializationVisitor;
+use JMS\Serializer\JsonSerializationVisitor;
```

Comment:
Same as above.

Reply:
did undo the change and still use the GenericSerializationVisitor until we upgrade